lib/routes/authorization.js

```diff
@@ -19,6 +18,13 @@ const verify = require('../browserid');
 const CODE = 'code';
 const TOKEN = 'token';
 
+const UNTRUSTED_CLIENT_SCOPES = [
+  'profile:uid',
+  'profile:email',
+  'profile:avatar',
```

Comment:
Indeed, "by default" has no meaning here.  We had some chat about sending a custom avatar just for pocket, but that's not in scope.

So yes, pocket will not get an avatar until we add a permissions page or decide to adjust this allowlist.